ktx/gradle.properties

```diff
@@ -0,0 +1 @@
+group=com.redmadrobot.extensions
```

Comment:
To understand: oh, gradle collects properties recursively?

Reply:
It should be, but it didn't :( Fixed it